test/parallel/test-net-connect-options-fd.js

```diff
@@ -33,8 +33,9 @@ const forAllClients = (cb) => common.mustCall(cb, CLIENT_VARIANTS);
 
 // Test Pipe fd is wrapped correctly
 {
-  const prefix = path.relative(`${common.PIPE}-net-connect-options-fd`,
-                               `${__dirname}-net-connect-options-fd`);
+  // Using relative path on osx if the path is greaterthan 108 chars
+  // an AssertionError: -48 is thrown
```

Comment:
OK, so maybe something like this is sufficiently non-specific but informative?

// Use relative path to avoid exceeding path length limit for sockets on some systems.